### ISSUE TITLE
English version for the loging page for facebook.

### ISSUE THREAD

**Issue by mckinon** [Cesar-Hack-Gray/FotoSploit#7]
For me FotoSploit works great 100% working but i need it to be in english to be more convincing for the loging page for facebook. If by chance you can do a more official page That would be great. Thanks again
![Screenshot_20200206-201831_Chrome](https://user-images.githubusercontent.com/59203782/73927970-16e25680-491e-11ea-952c-df261da277db.jpg)

**Comment by Cesar-Hack-Gray** [comment]
yes, in FotoSploit premium it already contains a Facebook in English friend

**Comment by mckinon** [comment]
My friend thank very much. Great job.

**Comment by supongba** [comment]
I can't add photo it says "No se pudo agregar la foto" 😥 please fix this.

**Comment by Cesar-Hack-Gray** [comment]
Hola te saluda Cesar Hack Gray

Verifica bien tu conexion a internet o fijate bien si ngrok correo al 100
en tu termux

**Comment by sslimbankx** [comment]
Hello Sir Cesar-Hack-Gray I already installed the fotosploit premium yet it is showing facebook spanish. What should I do please.
Thank you.

**Comment by Cesar-Hack-Gray** [comment]
Hola te saluda Cesar Hack Gray

Pronto subire una version ingles amigo


Att... Cesar

**Comment by sslimbankx** [comment]
Ok 
Thank you sir.
Can u notify me when you upload it?

**Comment by Cesar-Hack-Gray** [comment]
No te llegara amigo pero pondre la version 2.0 en el README.md amigo estate
al pendiente
